portal-api/test-helpers/mock-utilisation-report.ts

```diff
@@ -0,0 +1,28 @@
+import { UTILISATION_REPORT_RECONCILIATION_STATUS } from '@ukef/dtfs2-common';
+import { UtilisationReportResponseBody } from "../src/v1/api-response-types";
+import MOCK_BANKS from './mock-banks';
+
+export const aUtilisationReportResponse = (): UtilisationReportResponseBody => ({
```

Comment:
I really dislike the `MOCK_<>` constant pattern, and am very much an advocate for this way of writing test data. 
- None of them are actually _constant_ since you could easily just edit them in a test but their names and the way they are used implies they will be. 
- Using constants requires devs to know they shouldn't edit them in their tests, but if we do this instead then you don't have to worry about the way in which you edit your test objects and can edit them safely and freely at your will.
- On a slightly more pedantic note, I don't really like this use of the word 'mock', the `MOCK_<>`s are just instances of the classes in question, there isn't really any mocking going on.

The `MOCK_<>` pattern clearly makes sense for local dev data (and e2e tests possibly but I'm not too familiar with these yet) where we need to have constants that link with each other and we refer to in lots of places and do want the same object, but that doesn't hold for unit and api tests. Since we have so many different patterns as you point out, I would much rather go with and stick to this pattern going forwards as I do think it is better.

Reply:
That's all fair and I do agree that this is a better pattern. I just tend to just be wary of bringing in new patterns on this project, but if you think it's worth it then I'm happy for us to start using it